src/js/themes/base.js

```diff
@@ -1119,11 +1119,17 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         },
         width: '2px',
       },
+      background: {
+        color: {
+          dark: 'light-4',
+          light: 'light-1',
+        },
+      },
       check: {
         radius: '100%',
-        // background: {
-        //  color: undefined,
-        // },
+        background: {
+          color: undefined,
+        },
```

Comment:
Could this be left commented out and then just check if it is defined before using it in code?